trulens_eval/trulens_eval/tru.py

```diff
@@ -389,6 +390,47 @@ def Virtual(
 
         return TruVirtual(tru=self, app=app, **kwargs)
 
+    def BenchmarkExperiment(
+        self,
+        app_id: str,
+        ground_truth: Union[List, Callable, FunctionOrMethod],
+        trace_to_score_fn: Callable,
```

Comment:
Is there a reason this is called this when we call these things feedback functions everywhere else?

Reply:
originally this was meant for sth else and I wanted it to be general. but you're right - renamed back to feedback_fn now 👍